src/main/java/aquality/appium/mobile/configuration/ISwipeConfiguration.java

```diff
@@ -2,8 +2,78 @@
 
 import java.time.Duration;
 
+/**
+ * Describes Swipe settings.
+ */
 public interface ISwipeConfiguration {
+    /**
+     * Gets number of retries to perform swipe.
+     *
+     * @return arguments map.
+     */
     int getRetries();
 
+    /**
+     * Gets the timeout to perform a swipe.
+     *
+     * @return timeout to perform swipe.
+     */
     Duration getTimeout();
+
+    /**
+     * Gets the coefficient to calculate the 'x' coordinate of top point for swipe action.
+     *
+     * @return coefficient to calculate the 'x' coordinate of top point for swipe action
+     */
+    double getHorizontalSwipeTopPointXCoefficient();
+
+    /**
+     * Gets the coefficient to calculate the 'x' coordinate of bottom point for swipe action.
```

Comment:
why do we have to use the coefficient?  Is it the same for different devices? If not perhaps more simple is to define here  simple ''x' coordinate of bottom point'

please, leave more details about this

Reply:
There are dozens of devices with different screen height and width.
So using Appium we are only able to get the bottom right point of the screen.
So we need those coefficients to calculate end and start points for swipe actions relatively the right bottom corner.
Moved those coefficients into the configuration file, so that user could adjust coefficients to their application.
Does that answer your question?